### ISSUE TITLE
Empty string renders as missing translation

### ISSUE THREAD

**Issue by ruisilva450** [robisim74/angular-l10n#319]
**Describe the bug**
I have a typical json being loaded by `HttpTranslationLoader` in which a token leads to an empty string on that json

**To Reproduce**
1. Have a return json that looks like this:
```
{
    'textToTranslate': ''
}
```
2. Have a view that tries to translate 'textToTranslate'

**Expected behavior**
See `` (basically show nothing) instead of `textToTranslate`

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version 10.1.1

**Additional context**
You can reproduce this on the live demo.
[Here is the live demo with `changeLocale` value set as ""](https://stackblitz.com/edit/angular-l10n-lfu1a6)

**Comment by ruisilva450** [comment]
As a workaround I did this:
```
@Injectable()
export class L10nCustomMissingTranslationHandler implements L10nMissingTranslationHandler {
  private translation: L10nTranslationService;

  constructor(@Optional() private injector: Injector) {}

  public handle(key: string): string | any {
    this.translation = this.injector.get(L10nTranslationService);

    const data = this.translation.data;

    let value = null;
    try {
      value = key.split('.').reduce((a, v) => a[v], data[this.translation.getLocale().language]);
    } catch {}

    if (value != null) {
      return value;
    }
    return key;
  }
}
```

**Comment by robisim74** [comment]
Hi @ruisilva450,

In my opinion, an empty string is a missing translation.

But because this need has already been reported in the past, I will add `value` parameter to `handle` method, so everyone can handle it.

**Comment by ruisilva450** [comment]
For some locales you can have a use case where you don't want to show certain information and the way some people do it is by putting an empty string. 

Therefore my case

**Comment by robisim74** [comment]
I released a new version (10.1.2). Now you should be able to do this:

```
@Injectable() export class AppMissingTranslationHandler implements L10nMissingTranslationHandler {

    public handle(key: string, value?: string): string | any {
        if (value !== null) return value;
        return key;
    }

}
```

Let me know.

**Comment by ruisilva450** [comment]
Perfect.
